docs/src/RunningOnGPUs.md

```diff
@@ -60,9 +60,10 @@ The table below summarizes each backend's current capabilities.
 |                              | CUDA | ROCm   | Metal  | oneAPI |
```

Comment:
Maybe instead of the current design it would be better to add a new `cuTENSOR` row:
```suggestion
|                              | CUDA | cuTENSOR | ROCm   | Metal  | oneAPI |
```
where we can fill with `✓ (cuLAPACK)` for the factorizations. The idea could be that the table tells users that, when they load a package extension, if those operations are implemented and additional information about how the operations are run where relevant.

We could also specify `✓ (cuBLAS)` for `CUDA` `Contractions` and `✓ (cuLAPACK)` for `CUDA` factorizations.

Reply:
@mtfishman Sure I can do this. Should I write something up about how when you enable `cuTENSOR` you cannot revert to using `cuBLAS` for dense contractions? And that the cuTENSOR library is an extension to the CUDA.jl library?

Reply:
Sure, a short summary about `cuTENSOR` would be reasonable.